src/v0/destinations/customerio/data/customerIoGroup.json

```diff
@@ -28,7 +28,7 @@
   },
   {
     "destKey": "email",
-    "sourceKeys": ["context.traits.email", "properties.email", "context.externalId.0.id"],
+    "sourceKeys": ["context.traits.email", "properties.email", "traits.email"],
```

Comment:
This could be breaking change, can we confirm with @mihir-4116 once ?

Reply:
Yes done! and made the changes accordingly

Reply:
```javascript
"sourceKeys": ["context.traits.email", "properties.email", "traits.email", "context.externalId.0.id"],
```
Do we really need `traits.email` for hotfix ?

Reply:
I wouldn't put `traits.email` in the hotfix as it is something that is not part of the hotfix but rather needs a broader discussion with the team

Reply:
Removed